packages/mineral-ui-tokens/tokens/generic/borderWidth.json

```diff
@@ -0,0 +1,11 @@
+{
+  "global": {
+    "type": "pixel",
+    "category": "border"
+  },
+  "props": [
+    { "name": "borderWidth_medium", "value": "1px" },
```

Comment:
Looks like Mineral UI convention is:
```
  xxs
  xs
  sm
  md
  lg
  xl
  xxl
```

As seen in some of their constants files.  Let's use the same convention.